tables.go

```diff
@@ -238,13 +239,19 @@ func BatchInsertInTables(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
-
-	sql := config.PrestConf.Adapter.BatchInsertSQL(database, schema, table, names, placeholders[0])
-	sc := config.PrestConf.Adapter.BatchInsert(sql, values)
+	var sc adapters.Scanner
+	method := r.Header.Get("Prest-Batch-Method")
+	if strings.ToLower(method) != "copy" {
+		sql := config.PrestConf.Adapter.InsertSQL(database, schema, table, names, placeholders)
```

Comment:
`names` here is a `[]string` as say [travis](https://travis-ci.org/prest/controllers/jobs/374068936#L570)

Reply:
depend prest/adapters#42 signiature is change